pom.xml

```diff
@@ -321,7 +321,7 @@
         </identity.application.auth.adapter.package.export.version>
 
         <servlet-api.version>2.5</servlet-api.version>
-        <carbon.identity.framework.version>7.8.3</carbon.identity.framework.version>
+        <carbon.identity.framework.version>7.8.50-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
why this is updated to snapshot ?
Is it to run the tests ?

Reply:
Yes, once framework version released, will update with new version